### ISSUE TITLE
PostgreSQL migration Tests

### ISSUE THREAD

**Issue by ChuckKollar** [hubmapconsortium/antibody-api#159]
Reach out to Chris to get the new information regarding to the PostgreSQL that was recently upgraded.

Run tests on this repo as well as spatial which uses this database.

**Comment by ChuckKollar** [comment]
Zhou Yuan  [Dec 7, 23 10:03 AM]

I reached out and asked for [the slides used for the Demo Days presentation](https://docs.google.com/presentation/d/1ONpYdwXDufRnF8xPZM2SR-3bNUsiBMBOGVUwOCNiCVM/edit#slide=id.g2a2a39a3e35_1_77) apparently they forgot to upload, slide No.12 highlights your AVR.

Please reach out to @Chris Burke to obtain the database connection info for the migrated antibody-api (AVR) and spatial-api on DEV. We set up a new Postgres server in docker. Thanks to Chris, he moved the databases from AWS (for AVR only) and another separate spatial-api database to this new server. I've updated the API config files and both seem still work fine. We'll need your help to verify again, maybe upload some AVR reports to be sure. I don't know how to test the spatial-api calls. No rush, whenever after you finish other tasks.

**Comment by ChuckKollar** [comment]
From Joe,
We've been using two AWS RDS Postgres clusters one for DEV and one for PROD.
The antibody-api (AVR) connects to it. We had to reduce some AWS cost by downgrading EC2 a couple
of months ago, now it's time for the Postgres cluster since we really don't need a cluster for the AVR.
On the other hand, the spatial-api connects to a standalone docker container of Postgres that you put together long time ago to support some GIS extensions, because AWS didn't support extensions back
then. So recently, I set up a standalone Postgres container using the same dockerfile, to support both AVR and spatial-api. Chris created individual databases for each and had migrated the DEV data to this
new server. I had updated the api config to connect to each new database too.

We need your final review to double check before moving to PROD.
